Interop/StubGenerator/src/main/kotlin/org/jetbrains/kotlin/native/interop/gen/ObjCStubs.kt

```diff
@@ -65,195 +64,7 @@ private fun ObjCMethod.getFirstKotlinParameterNameCandidate(forConstructorOrFact
     return this.parameters.first().name?.takeIf { it.isNotEmpty() } ?: "_0"
 }
 
-private fun ObjCMethod.getKotlinParameters(
-        stubGenerator: StubGenerator,
-        forConstructorOrFactory: Boolean
-): List<KotlinParameter> {
-    val names = getKotlinParameterNames(forConstructorOrFactory) // TODO: consider refactoring.
-    val result = mutableListOf<KotlinParameter>()
-
-    this.parameters.mapIndexedTo(result) { index, it ->
-        val kotlinType = stubGenerator.mirror(it.type).argType
-        val name = names[index]
-        val annotations = if (it.nsConsumed) listOf("@CCall.Consumed") else emptyList()
-        KotlinParameter(name, kotlinType, isVararg = false, annotations = annotations)
-    }
-
-    if (this.isVariadic) {
-        result += KotlinParameter(
-                names.last(),
-                KotlinTypes.any.makeNullable(),
-                isVararg = true,
-                annotations = emptyList()
-        )
-    }
-
-    return result
-}
-
-class ObjCMethodStub(private val stubGenerator: StubGenerator,
```

Comment:
It makes a lot of sense to keep reworked generator pieces in its places.
This would help to keep searching through git simple and to avoid placing all code to the single file.